docs/pages/inboxes/build-inbox.md

```diff
@@ -0,0 +1,172 @@
+## Build the inbox
+
+:::warning[🚧 XMTP V3 Alpha]
```

Comment:
Open to feedback on this callout. I know that the XMTP v3 group chat SDK features are used in the Converse production app - so calling about XMTP V3 as alpha seems inaccurate.
 
However, I was thinking that if we are talking about building an app with both DMs and group chat on V3 - this is what could qualify this as a beta?

Also, if we are talking about an existing app built with V2 - starting to build DMs and group chat on V3 - this could qualify as a beta in that we still need to define a recommended migration path.

Again - open to all feedback on this!